docs/manifesto.md

```diff
@@ -0,0 +1,47 @@
+# Shaken Fist Manifesto
+
+This document attempts to list Shaken Fist's defining features, give guidance on what type of features should be added to the project, how they should be implemented and how we work together.
+
+## Shaken Fist Defining Characteristics
+
+* Shaken Fist is smaller, simpler cloud.
+* It is designed for relatively small environments with minimum management overhead.
+* Its features are highly opinionated. This means that the maintainers have chosen the best (in their opinion) features to support.
+* Opinionated features do not handle every single possible use case. This reduces the code base size thus increasing long-term maintainability.
+* The code base is understandable in its entirety by a single developer.
```

Comment:
Specifically, I have an emotional barrier at 5,000 lines of python for the core.

Reply:
Hmmmm
```
and@baldr:~/sf/shakenfist/shakenfist (master)$ cloc  .
      42 text files.
      42 unique files.                              
       8 files ignored.

github.com/AlDanial/cloc v 1.82  T=0.14 s (273.6 files/s, 65162.1 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                          37           1572            356           6884
-------------------------------------------------------------------------------
SUM:                            37           1572            356           6884
```

Reply:
Is that including tests?

Reply:
sshhhh... Looking only for core code this time.
```
and@baldr:~/sf/shakenfist/shakenfist (master)$ cloc  --exclude-dir=tests,client .
      24 text files.
      24 unique files.                              
       2 files ignored.

github.com/AlDanial/cloc v 1.82  T=0.03 s (934.9 files/s, 225615.8 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                          24           1174            318           4300
-------------------------------------------------------------------------------
SUM:                            24           1174            318           4300
-------------------------------------------------------------------------------
```
 (Maybe we should break it up into a few different repositories?? :sunglasses:  )

Reply:
Yeah, I exclude tests. We'll one day exceed that, but for now it makes me happy.